public/controllers/agent/components/register-agent-service.ts

```diff
@@ -17,22 +17,47 @@ type RemoteConfig = {
   haveSecureConnection: boolean | null;
 };
 
+/**
+ * Get the cluster status
+ */
+export const clusterStatusResponse = async (): Promise<boolean> => {
+  const clusterStatus = await WzRequest.apiReq('GET', '/cluster/status', {});
```

Comment:
Should this request handle possible API errors?

Reply:
The errors are controlled in this function

https://github.com/wazuh/wazuh-kibana-app/blob/4.4-7.16/public/controllers/agent/register-agent/steps/server-address.tsx#L39-L63